src/migrations/00072-api-key-expiry.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "api key" ADD COLUMN "expiry date" TIMESTAMP NULL;
```

Comment:
I don't think this should be necessary as there should be a migration built in to pinejs already?

Reply:
Same as above. Mainly for debugging. Removed now.